resources/aws/peering.go

```diff
@@ -0,0 +1,127 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	RequesterVpcFilterName = "requester-vpc-info.vpc-id"
+	AccepterVpcFilterName  = "accepter-vpc-info.vpc-id"
+)
+
+type VPCPeeringConnection struct {
+	VPCId     string // VPCId is the ID of the VPC in the guest cluster.
```

Comment:
Nitpicky but we're mainly using VpcID in other places. I'd rather not introduce another casing variant.

Reply:
We should then fix the casing because based on go conventions the proper case is `VPC`, not `Vpc`.

Reply:
Got it and yes we should fix the existing casing then.

Reply:
The casing has been driving me nuts all over :D